cmd/kubectl-gadget/utils/trace.go

```diff
@@ -379,11 +393,9 @@ func SetTraceOperation(traceID string, operation string) error {
 	return err
 }
 
-// waitForOutput loops over all trace whom ID is given as parameter waiting
-// until they are in the expected state.
-// After this function and if correct state was given as parameter, the trace
-// output should contain the needed information.
-func waitForOutput(traceID string, expectedState string) (gadgetv1alpha1.TraceList, error) {
+// waitForTraceInState loops over all trace whom ID is given as parameter
+// waiting until they are in the expected state.
+func waitForTraceInState(traceID string, expectedState string) (gadgetv1alpha1.TraceList, error) {
```

Comment:
Does the name `waitForTraceInState` come from "wait for trace initial state"?

IMO, `waitForTraceState` would be more appropriate given that it could wait for both `Initial` and `Output` states. WDYT?

Reply:
`waitForTraceInState` was for "wait for trace in state".
But I agree with your suggestion.